modules/wallet/seed_test.go

```diff
@@ -108,6 +108,8 @@ func TestLoadSeed(t *testing.T) {
 		t.Error("AllSeeds returned the wrong seed")
 	}
 
+	wt.wallet.db.Close() // db must be closed before reuse
```

Comment:
it's because the test calls `New` in the same directory twice. Without the `Close`, you'd get a timeout error when trying to open the database.
I first tried `wt.wallet.Close`, but sometimes the test wants to compare one wallet to another, which it can't do after `Close` has been called.